docs/configuration.md

```diff
@@ -24,7 +24,8 @@ Some less common environment variables are:
 | `GHE_PROTOCOL`        | `new Probot({ Octokit: ProbotOctokit.defaults({ baseUrl }) })`        | The protocol of your GitHub Enterprise instance. Defaults to HTTPS. Do not change unless you are certain. <p>_(Example: `https`)_</p>                                                                                                                                                                       |
 | `LOG_FORMAT`          | -                                                                     | By default, logs are formatted for readability in development. You can set this to `json` in order to disable the formatting                                                                                                                                                                                |
 | `LOG_LEVEL`           | `const log = require('pino')({ level })`                              | The verbosity of logs to show when running your app, which can be `fatal`, `error`, `warn`, `info`, `debug`, `trace` or `silent`. Default: `info`                                                                                                                                                           |
-| `LOG_LEVEL_IN_STRING` | -                                                                     | By default, when using the `json` format, the level printed in the log records is an int (`10`, `20`, ..). This option tells the logger to print level as a string: `{"level": "info"}`. Default `false`                                                                                                    |
+| `LOG_LEVEL_IN_STRING` | -                                                                     | By default, when using the `json` format, the level printed in the log records is an int (`10`, `20`, ..). This option tells the logger to print level as a string: `{"level": "info"}`. Default `false`
+| `LOG_MESSAGE_KEY` | -                                                                     | By default, when using the `json` format, the string key for the 'message' in the JSON object, E.g., `{"message": "My logging message"}`. Default `msg`                                                                                                    |
```

Comment:
```suggestion
| `LOG_MESSAGE_KEY` | -                                                                     | Only relevant when `LOG_FORMAT` is set to `json`. Sets the json key for the log message. Default: `msg`                                                                                                    |
```

Reply:
I cannot commit your suggestion for some reason.